app/assets/src/components/views/discovery/DiscoverySidebar/DiscoverySidebar.jsx

```diff
@@ -105,7 +105,7 @@ export default class DiscoverySidebar extends React.Component {
       <div className={cs.histogramContainer}>
         <div className={cs.dateHistogram}>
           {dates.map(entry => {
-            const percent = Math.round(100 * entry.count / total, 0);
+            const percent = Math.round((100 * entry.count) / total, 0);
```

Comment:
Why the extra parentheses?

Reply:
I saw that there were unintentional changes to the files I edited and just changed the formatting of some code... am I missing a linter? Not sure why those lines were changed

Reply:
Could be... or for some reason you are running a different linter version (globally installed?)  that has different default rules.